benchmark/src/main/scala/org/virtuslab/akkasaferserializer/benchmark/Main.scala

```diff
@@ -0,0 +1,44 @@
+package org.virtuslab.akkasaferserializer.benchmark
+
+object JavaSerialization {
+  def main(args: Array[String]): Unit = {
+    Benchmark.benchmark("application-java.conf", "json/java.txt")
+    System.exit(0)
```

Comment:
🤔 why explicitly?

Reply:
Is it b/c of an Akka system still running in the background?

Reply:
exactly, Akka leaves some background thread that keeps the application running